### PR TITLE
add static server,websocket refresh watching mode

### PR DESCRIPTION
**Pull request by bung87**
helpful during dev

### CODE REVIEW

File: karax/tools/karun.nim [hunk 1]
```diff
@@ -73,25 +111,10 @@ proc main =
     of cmdEnd: break
 
   if file.len == 0: quit "filename expected"
-  let name = file.splitFile.name
-  build(name, rest, selectedCss, run)
-  if watch:
```
Comment (Araq) [inline karax/tools/karun.nim:78]: Watching shall remain optional please.

**Comment by Araq** [comment]
Great addition! But please see my remark.

File: karax/tools/karun.nim (hunk 1 above)
Comment (bung87) [reply karax/tools/karun.nim:78]: watching optional now.

File: karax/tools/karun.nim [hunk 2]
```diff
@@ -73,25 +117,10 @@ proc main =
     of cmdEnd: break
 
   if file.len == 0: quit "filename expected"
-  let name = file.splitFile.name
-  build(name, rest, selectedCss, run)
+  spawn serve()
```
[the remark this answers is not in the document]
Comment (bung87) [reply karax/tools/karun.nim:120]: so server also optional ?

[the remark this answers is not in the document]
Comment (Araq) [reply karax/tools/karun.nim:120]: Yes please.

**Comment by bung87** [comment]
might be able ignore some files during watching , with extensions like .scss .less etc.

**Pull request merged**